packages/framework/src/Form/ImageUploadType.php

```diff
@@ -124,6 +126,39 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'accept' => ImageProcessor::SUPPORTED_IMAGE_MIME_TYPES,
             ],
         ]);
+
+        $builder
+            ->add('uploadedFilenames', CollectionType::class, [
+                'entry_type' => LocalizedType::class,
+                'allow_add' => true,
+                'entry_options' => [
+                    'label' => '',
+                    'entry_options' => [
+                        'constraints' => [
+                            new Assert\Length([
+                                'max' => 245,
+                                'maxMessage' => 'File name cannot be longer than {{ limit }} characters',
+                            ]),
+                        ],
+                    ],
+                ],
+            ])->add(
+                $builder->create('currentFilenamesIndexedByImageIdAndLocale', CollectionType::class, [
```

Comment:
```suggestion
                $builder->create('namesIndexedByImageIdAndLocale', CollectionType::class, [
```
in the task specification it is ALT

Reply:
Yes, it is alternative for image, when it is not loaded. We use name for this by default.